config/clusters/leap/enc-prod.secret.values.yaml

```diff
@@ -0,0 +1,22 @@
+basehub:
+    jupyterhub:
+        hub:
+            config:
+                GitHubOAuthenticator:
+                    client_id: ENC[AES256_GCM,data:auopWrLSGIBDtQ4PrZZMYe5XFWM=,iv:+tzBIkE6R3PfJm7oYyJOq84yyD6tB3GXeQ++sYPU7S8=,tag:vQrhczBhRRaFoqqwRWeGHg==,type:str]
+                    client_secret: ENC[AES256_GCM,data:xLL5GJTKSnucssmIQjVhCUwwXyZaYl54/+QzXPFx0dJpX63kaeJufw==,iv:2cyHZvDaoNQtlKiPKf2ACoNuvlww6WE7vcGG6jVXISI=,tag:IRcogW5ZDZWA6Pv8DyVcPg==,type:str]
+                    oauth_callback_url: ENC[AES256_GCM,data:d+/oCcmELV7Tvfe86P4YH8DCnLHI0yid0WoUWH0IKT022b9Ba/rnptYp,iv:SVHQK5yK26JOHV6uWycsLYUk42g6Kl8RahOf5oMbqxc=,tag:HjT+HYo30qqP5yCKcgVZCQ==,type:str]
```

Comment:
I'm in two minds about having oauth_callback_url here. I know it makes the number of files you need to pass around nicer, but this value doesn't need to be encrypted really. And I did lose some time thinking that the uwhackweek hubs were misconfigured because of this pattern before.

No specific action to take in this PR, but I'd like to discuss as a team some guidelines around this for readability's sake.

Reply:
@sgibson91 yeah I was thinking about that too. While keeping the number of files down helps, I think keeping it separate works alright for me too. But in that case I want to split it into *three* files - common, staging and prod, rather than just staging and prod. How does that sound?

Reply:
Sounds reasonable. The deployer will work regardless, so long as they're listed in the cluster.yaml file! :tada:

Reply:
@sgibson91 i've split this now into two non-secret files staging.yaml + prod.yaml.